js/index.js

```diff
@@ -1 +1,74 @@
 // Your code goes here
+
+//Header
+const navItems = document.querySelectorAll("nav a");
+const copyright = document.querySelector(".footer .hidden");
+//when "contact" is double clicked, copyright will appear/disappear
+navItems[3].addEventListener("dblclick", event => {
+  copyright.classList.toggle("hidden");
```

Comment:
```suggestion
  event.preventDefault();
  copyright.classList.toggle("hidden");
```
Prevent default to keep page from going to the contact link. Since it links back to this page it refreshes the page, this will stop that.